libs/features/cdt/cdt.css

```diff
@@ -35,6 +35,12 @@
   display: flex;
 }
 
+html[dir="rtl"] .horizontal .timer-block {
+  margin-right: 10px;
```

Comment:
let's avoid using left or right. Instead use start/end

Reply:
By using the suggestion I made below you can remove this selector.

Reply:
I have achieved it using column-gap property.
Selector can't be removed, we need "flex-direction: row-reverse;" for rtl